reana_workflow_engine_snakemake/executor.py

```diff
@@ -250,15 +251,26 @@ def run_jobs(
     operational_options={},
 ):
     """Run Snakemake jobs using custom REANA executor."""
+    workflow_file_path = os.path.join(workflow_workspace, workflow_file)
+    common_snakemake_args = dict(
+        snakefile=workflow_file_path,
+        config=workflow_parameters,
+        workdir=workflow_workspace,
+        keep_logger=True,
+        # Since Snakemake v7.3.0, if default resources are not specified when in
```

Comment:
A suggestion for the comment:

```
        # Since Snakemake v7.3.0, the workflow logs include Snakemake-percieved native
        # resource information on memory and storage (`mem_mb`, `disk_mb`) for each
        # Snakemake rule. However, REANA overrides these when running user jobs, so we
        # should hide them in order not to present any misleading information to users.
        # For this reason, the default resources are overridden here with the only the
        # "bare" ones (`tmpdir`).
```